setup.cfg

```diff
@@ -84,10 +84,14 @@ optional =
 doc =
```

Comment:
I'd say to leave these out of `etc/environment.yml` as only a few developers care to build docs. But it could be worth expanding in DEVELOPER.md. Perhaps I'll amend this PR....

Reply:
mentioned the `doc` group in `DEVELOPER.md` in #1604